test/avi/t_insert_mode.clj

```diff
@@ -130,6 +130,8 @@
                     "~" :blue
                     "test.txt" :black :on :white
                     ""])
+  (fact "repeat count for `i` with `3ix<Esc>` leaves the point on last `x` added"
```

Comment:
Thanks for the inputs Jason Felice.

I found the cause of problem. Point is not updated after the recursive
reduce calls. But how to ensure point is updated I am not having an answer
at this point. Still fighting with code :)

I shall do the changes you pointed out. Is there something else you suggest
I can work on?

Regards,

Arun SL

On 09-Sep-2016 8:11 pm, "Jason Felice" <notifications@github.com> wrote:

In test/avi/t_insert_mode.clj
<https://github.com/maitria/avi/pull/23#discussion_r78191727>:

> @@ -130,6 +130,8 @@
>                      "~" :blue
>                      "test.txt" :black :on :white
>                      ""])
> +  (fact "repeat count for `i` with `3ix<Esc>` leaves the point on last `x` added"

If you want to add a test but aren't ready to implement it, you can replace
the word fact with the word future-fact, and it won't fail the build. It
will note there's something left to do.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/maitria/avi/pull/23/files/13a011936045835f5a42939940679ce5cf0c1858#r78191727>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AFtf6n9dpIabTjXCypI-TE3VsLEJ3zYVks5qoXAkgaJpZM4J40il>
.